girder/utility/model_importer.py

```diff
@@ -17,44 +17,34 @@
 #  limitations under the License.
 ###############################################################################
 
-import importlib
-import six
+_modelClasses = {}
+_coreModelsRegistered = False
```

Comment:
This is another case of global context that might cause issues in testing.  Should we reset it in the db fixture?

Reply:
Not sure. We don't currently have any tests that depend on the core models being unregistered, so I'm inclined to think YAGNI, but I could go either way.